src/utils.py

```diff
@@ -58,6 +54,26 @@ def find_packages_by_indices(indices, base=""):
     return packages
 
 
+def _get_archive_root(pool: Path) -> Path:
+    """Get arive path from pool path."""
```

Comment:
arive -> archive

Reply:
Addressed these typos in [#28](https://github.com/canonical/charm-apt-mirror/pull/28).